{{ cookiecutter.project_slug }}/Makefile

```
@@ -172,3 +175,22 @@ run-acceptance-tests: ## Run Acceptance tests
 	npx wait-on --httpTimeout 20000 http-get://localhost:55001/plone http://localhost:3000
 	$(MAKE) -C "./frontend/" test-acceptance-headless
 	$(MAKE) stop-acceptance-servers
+
+## Acceptance Frontend tests in development mode
+.PHONY: start-test-acceptance-frontend-dev
+start-test-acceptance-frontend-dev: ## Start the Acceptance Frontend Fixture in development mode
+	(cd frontend && RAZZLE_API_PATH=http://127.0.0.1:55001/plone yarn start)
+
+.PHONY: start-test-acceptance-server
+start-test-acceptance-server: build-acceptance-servers ## Start Backend Acceptance Servers in development mode
+	@echo "Start acceptance backend"
+	@docker run --rm -p 55001:55001 --name {{ cookiecutter.project_slug }}-backend-acceptance -d {{ cookiecutter.github_organization }}/{{ cookiecutter.project_slug }}-backend:acceptance
+
```

Let's add more user feedback, and if the docker command fails, then it won't echo the next command, which can be helpful in troubleshooting.

```suggestion
	@echo "Started test acceptance server"

```